rkt/run.go

```diff
@@ -33,17 +33,21 @@ import (
 
 var (
 	cmdRun = &cobra.Command{
-		Use:   "run [--volume=name,kind=host,...] IMAGE [-- image-args...[---]]...",
+		Use:   "run [--volume=name,kind=host,...] [--mount volume=VOL,target=PATH] IMAGE [-- image-args...[---]]...",
 		Short: "Run image(s) in a pod in rkt",
 		Long: `IMAGE should be a string referencing an image; either a hash, local file on disk, or URL.
 They will be checked in that order and the first match will be used.
 
+Volumes are made available to the container via --volume.
+Mounts bind volumes into each image's root within the container via --mount.
+--mount is position-sensitive; occuring before any images applies to all images,
+occuring after any images applies only to the nearest preceding image.
```

Comment:
Maybe mention that per-app mount point takes precedence (or overrides) the pod mount point with the same path?